docs/user/document/acknowledgments-usage.en.md

```diff
@@ -1 +1,9 @@
 # Acknowledgments - Usage
+
+Acknowledgments of people and organizations that were involved in discovery, publication and coordination of the process that lead to the CSAF document.
+
+The acknowledgments are an important tool to reward and honor security researchers and contribute to the reputation of an organization.
+
+In contrast to the Vulnerability Acknowledgments (`vulnerabilities[]/acknowledgments`) contributions to the whole document can be honored.
```

Comment:
I would use the absolute notation: `/vulnerabilities[]/acknowledgments`

Reply:
updated - also at some other similar places, see affd014b

Reply:
:+1: I like that you think for yourself! :smile: